junifer/data/parcellations.py

```diff
@@ -805,6 +908,9 @@ def _retrieve_suit(
     resolution = closest_resolution(resolution, _valid_resolutions)
 
     # define file names
+    # Format space if MNI; required for the file name
+    if space.startswith("MNI"):
```

Comment:
I would leave the right space and not just MNI in the filename. Even if this triggers a second download on old users.